languages/elixir/exercises/concept/language-list/.docs/instructions.md

```diff
@@ -14,25 +14,33 @@ LanguageList.new()
 Define the `add/2` function that takes 2 arguments (a _language list_ and a string literal of a _language_). It should return the resulting list with the new language prepended to the given list.
 
 ```elixir
-LanguageList.new() |> LanguageList.add("Clojure")
-# => ["Clojure"]
+LanguageList.new()
+|> LanguageList.add("Clojure")
+|> LanguageList.add("Haskell")
+# => ["Haskell", "Clojure"]
 ```
 
 ## 3. Define a function to remove a language from the list
 
-Define the `remove/2` function that takes 1 argument (a _language list_). It should return the list minus the first item.
+Define the `remove/2` function that takes 1 argument (a _language list_). It should return the list without the first item.
 
 ```elixir
-LanguageList.new() |> LanguageList.add("Haskell") |> LanguageList.remove()
-# => []
+LanguageList.new()
+|> LanguageList.add("Clojure")
+|> LanguageList.add("Haskell")
+|> LanguageList.remove()
+# => ["Clojure"]
```

Comment:
Or maybe it would be clearer to define the snippets using a literal list instead of the other functions?
```suggestion
LanguageList.remove(["Haskell", "Clojure"])
# => ["Clojure"]
```

Reply:
I think either is fine, I think at the time I just wanted to reinforce the usage of the exercise functions.